central/resourcecollection/service/service.go

```diff
@@ -0,0 +1,27 @@
+package service
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/central/resourcecollection/datastore"
+	v1 "github.com/stackrox/rox/generated/api/v1"
+	"github.com/stackrox/rox/pkg/grpc"
+)
+
+// Service provides the interface to the microservice that serves alert data.
+type Service interface {
+	grpc.APIService
+
+	AuthFuncOverride(ctx context.Context, fullMethodName string) (context.Context, error)
+
+	v1.CollectionServiceServer
+}
+
+// New returns a new Service instance using the given DataStore.
+func New(
```

Comment:
[nit] Is this an artifact of gofmt? If not, IMO it looks nicer on one line

Reply:
This was a copy/paste from another implementation that had multiple fields and the formatting never got changed after trimming down args. I'm going to see if the tests pass before I push this small change

Reply:
Not a big deal. You can combine this with another PR when you implement other endpoints.